test/OpenFeature.DependencyInjection.Tests/OpenFeatureBuilderExtensionsTests.cs

```diff
@@ -28,12 +27,11 @@ public void AddContext_Delegate_ShouldAddServiceToCollection(bool useServiceProv
             _systemUnderTest.AddContext((_, _) => { });
 
         // Assert
-        featureBuilder.Should().BeSameAs(_systemUnderTest, "The method should return the same builder instance.");
-        _systemUnderTest.IsContextConfigured.Should().BeTrue("The context should be configured.");
-        _services.Should().ContainSingle(serviceDescriptor =>
+        Assert.Equal(_systemUnderTest, featureBuilder);
```

Comment:
The message is lost here.
```suggestion
        Assert.Equal(_systemUnderTest, featureBuilder, "The method should return the same builder instance.");
```

Reply:
Xunit doesn't have an overload to allow you to specify a debug message

I could swap to use Assert.True instead, which allows us to keep the debug messages

```csharp
Assert.True(featureBuilder == _systemUnderTest, "The method should return the same builder instance.")
```

Reply:
You are correct. There is no overload for the `Assert.Equals`. @open-feature/sdk-dotnet-maintainers, should we change to do an `Assert.True` with a message, or should we keep the `Assert.Equals`?

I prefer to keep the message for later debug/test visibility, even if it looks a bit "hacky" due to the lack of overload.

Reply:
We lose some of the out of the box functionality that comes with `Assert.Equal` if we transition to `Assert.True`

I think we have several options:
1. Remove the debug messages (as is)
2. Switch to `Assert.True(predicate, message)`
3. Adopt [shouldly](https://github.com/shouldly/shouldly) (like you mention in https://github.com/open-feature/dotnet-sdk/issues/353)
4. Leave FluentAssertions but fix to v7

Happy to proceed with any of the above 😃

Reply:
I'm in favor of leaving it as is for now. It would solve the FluentAssertions issue and we could always move to shouldly (or something similar) later. Does that reasonable @kylejuliandev @askpt?

Reply:
@beeme1mr and @kylejuliandev, I think that is a fair approach.
I would like to keep the messages due to its importance so an effort to move to shouldly needs to be considered. Shouldly targets dotnet 8 and standard 2.0 (it shouldn't be a problem for .net framework 4.6+)

Reply:
Thanks both. I will resolve these comments and merge in any changes from main this afternoon.